packages/web/src/components/spa/HackAUBG/FAQSection/FAQModule.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useState } from 'react';
+import { AiOutlinePlus } from 'react-icons/ai';
+import { AiOutlineMinus } from 'react-icons/ai';
+import './faq_module.css';
+
+export const FAQModule = props => {
+    const [stepContent, setStepContent] = useState('faq-container-content-not-displayed');
+    // eslint-disable-next-line no-unused-vars
```

Comment:
Why is this comment here?

Reply:
No idea, most of the code was reused from last year's journey mobile section